prisma/migrations/20240423164711_form_purpose/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Template" ADD COLUMN     "formPurpose" TEXT NOT NULL DEFAULT '';
```

Comment:
Noting given the migration this won't be testable with the review environments given we use staging resources (which is fine just flagging).